doc/source/PBcount.rst

```diff
@@ -36,7 +37,7 @@ Example
 .. code-block:: bash
 
     $ PBcount -f psi_md_traj_1.PB.fasta -o psi_md_traj_1
-    read 90 sequences in psi_md_traj_1.PB.fasta
+    read 225 sequences in psi_md_traj_1.PB.fasta
```

Comment:
It's not the same trajectory anymore ?
We have 225 frames now in psi_md_traj_1
This change also the output images